EnrichStream/pom.xml

```diff
@@ -49,7 +49,8 @@ under the License.
         <dependency>
             <groupId>com.trifork.cheetah.processing</groupId>
             <artifactId>cheetah-lib-processing</artifactId>
-            <version>4.1.1</version>
+            <version>4.2.0-feature-lib-processing-image-SNAPSHOT</version>
```

Comment:
snapshot

Reply:
yes, i would like to merge example-flink first using the lib-processing snapshot and then lib-processing e2e tests will go through since they use the main branch of example-flink